rubygem-safe_yaml/rubygem-safe_yaml.spec

```diff
@@ -0,0 +1,87 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name safe_yaml
+
+Summary: Parse YAML safely
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.4
+Release: 1%{?dist}
+Group: Development/Languages
+License: MIT
+URL: https://github.com/dtao/safe_yaml
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix}ruby(rubygems)
+Requires: %{?scl_prefix}ruby
```

Comment:
These both need to be `scl_prefix_ruby`, otherwise they'll expand to tfm-ruby, which doesn't exist.